src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -16,8 +16,19 @@
 var ReactFiberReconciler = require('ReactFiberReconciler');
 var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
+var ReactTypeOfWork = require('ReactTypeOfWork');
```

Comment:
sorry.... can you explain what you want me to try here?  just run `npm run build`?